nodeup/pkg/model/ntp.go

```diff
@@ -17,10 +17,13 @@ limitations under the License.
 package model
 
 import (
+	"fmt"
+	"io/ioutil"
 	"k8s.io/klog"
 	"k8s.io/kops/nodeup/pkg/distros"
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kops/upup/pkg/fi/nodeup/nodetasks"
+	"regexp"
```

Comment:
This needs to be moved before the k8s ones.
Try also `make goimports`.

Reply:
Thanks @hakman I have moved It up before the k8 ones.  For some reason go imports is not working on my Mac.  Once I have fixed that I'll run it.